zerver/lib/actions.py

```diff
@@ -5407,6 +5407,11 @@ def do_create_realm(
     if enable_spectator_access is not None:
         kwargs["enable_spectator_access"] = enable_spectator_access
 
+    # Welcome emails are disabled for demo orgs since their owners
+    # may not have a valid email set.
```

Comment:
s/may/will/, right?  Also, I'd call it "demo organizations" -- part of our "no abbreviations" policy.

Reply:
Done!